3-create-support-files-and-dashboard-to-answer-bq-23

jdcastellanosb73 opened the pull request:
3-create-support-files-and-dashboard-to-answer-bq-23.

the task was completed with the .xls, the control board and a link to it

jdcastellanosb73 committed:
the tanks is finished with the controlboard

jsurrea commented on README.md:
```diff
@@ -1 +1,3 @@
\ No newline at end of file
+# Analytics
+
+link para el looker studio BQ 2.3: https://lookerstudio.google.com/s/j8pzo3OZxkg
```
Hey can you update this to be in English? Thank you!

jsurrea replied:
Also with the file names!

jsurrea replied:
Also, to ensure consistency, rename the folder to BQ 2.3

jsurrea replied:
Hey @jdcastellanosb73 ! Can you change the folder name (add the space) and file names (to English) as well?

jsurrea reviewed (changes requested):
Please remember to always use English!

jdcastellanosb73 committed:
README updated

jdcastellanosb73 committed:
update the pdf to show the BQ question

updated the pdf to show the BQ question

jdcastellanosb73 committed:
added the control board as image

the image of the control board is generated

jsurrea commented on README.md:
```diff
@@ -1 +1,3 @@
\ No newline at end of file
+# Analytics
+
+link for the looker studio to answer the BQ 2.3: https://lookerstudio.google.com/s/j8pzo3OZxkg
```
https://lookerstudio.google.com/s/j8pzo3OZxkg is not a public link! Check this:


Uploading Screen Recording 2024-03-22 at 16.03.09.mov…

SnowArtz reviewed (changes requested):
I think we should use a different folder structure, i would use meanwhile the next Folders:

excel_mockups
dashboards

And the name conventions of the excel: bqx.y.xlsx
And the name convention of the dashboard: bqx.y.pdf

jdcastellanosb73 committed:
folders created for organizations

folders for organization

jdcastellanosb73 committed:
change the name of the code used to generate the data

change names for order

SnowArtz reviewed (changes requested):
Please Upload a Python file instead a Jupyter Notebook!

jdcastellanosb73 committed:
link change to public

the link to the dashboard is changed to public

SnowArtz reviewed (changes requested):
Please Add a small text into the upper left related to your BQ number: 2,3

jdcastellanosb73 committed:
change link to information

-change in link to make it public
-change BQ format to pdf
- change format of generator from ipynb to py

jdcastellanosb73 committed:
change name of BQ in dashboard

change the name to the correct one

SnowArtz reviewed (changes requested):
You havent delete the notebook.

jdcastellanosb73 committed:
tittle to charts on dashboard and ipynb deleted

delete the ipynb and tittles on the dashboards charts

SnowArtz commented on README.md:
```diff
@@ -1 +1,3 @@
\ No newline at end of file
+# Analytics
+
+link for the looker studio to answer the BQ 2.3: https://lookerstudio.google.com/reporting/9281694c-2631-46f5-8aab-0b23cb568ee9
```
This link now is with editor permissions!! Take care,

Laudarias reviewed (approved):
Link is working correctly now

SnowArtz reviewed (approved):
Nice!

SnowArtz committed:
Merge branch 'develop' into 3-create-support-files-and-dashboard-to-answer-bq-23
